packages/react-native-bpk-component-rating/src/BpkRating.js

```diff
@@ -26,19 +27,36 @@ const arrify = (maybeArray: Array<string> | string): Array<string> =>
   Array.isArray(maybeArray) ? maybeArray : [maybeArray];
 
 const BpkRating = (props: Props) => {
-  const { title, subtitle, icon, ...rest } = props;
+  const {
+    title,
+    subtitle: any,
```

Comment:
This seems like a confusing name 🤔

Reply:
Why?

Reply:
I thought "any" was the name, but now I realise it was a type annotation... 😆

Reply:
Np, it did remind me to remove it